addon/config.json

```diff
@@ -0,0 +1,37 @@
+{
+	"name": "Leaf2MQTT",
+	"version": "37",
```

Comment:
Can you put `latest` here instead to always fetch the latest version so we don't have to update this in each commits?

Reply:
Something like this (untested).

```
  update_addon:
    name: Update Home Assistant add-on
    needs: push_to_registry
    runs-on: ubuntu-latest
    steps:
      - name: Checkout code
        uses: actions/checkout@v2

      - name: Change image version number
        uses: jossef/action-set-json-field@v1
        with:
          file: addon/config.json
          field: version
          value: ${{ github.run_number }}

      - name: Commit changes
        uses: stefanzweifel/git-auto-commit-action@v4
        with:
          commit_message: Update add-on version
```

Reply:
Ok no problem, as long as the config.json is not added to the current workflow trigger paths to prevent circular updates

Reply:
It is in the new add-on folder, which should not trigger the workflow.
Workflows are new to me, I think the code above should work, but I don't know how to test it without triggering an image update. Let me know if you want me to add it to the PR.

Reply:
to test it, you can fork this repository, remove the existing job (since you won't have access to dockerhub secret) and hardcode the `${{ github.run_number }}` to an non existing version  and comment the  `needs: push_to_registry` then add a comment somewhere in the code to provoke a build on your fork. If this does not work to test it then let me know, we will find another solution

Reply:
I also don't mind if you add it to your PR